java/src/main/java/org/rocksdb/UInt64AddOperator.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
You need to have the other license related lines too after this copyright line. So, could you please add back:
```
//  This source code is licensed under both the GPLv2 (found in the
//  COPYING file in the root directory) and Apache 2.0 License
//  (found in the LICENSE.Apache file in the root directory).
```

Reply:
Thanks for taking care of this @ajkr .